frontend/app/api/airstack/followers/route.ts

```diff
@@ -0,0 +1,43 @@
+import { fetchQuery, init } from "@airstack/node";
+
+if (!process.env.AIRSTACK_TOKEN) throw new Error("AIRSTACK_TOKEN is not set in environment variables");
+
+init(process.env.AIRSTACK_TOKEN);
+
+const getSocialFollowersQuery = `query GetSocial($identity: Identity!) {
```

Comment:
I think I will move these calls to the `UpdateSocialData` service and store the info in our database, so we only have to fetch the info once. We can allow the user to resync this info in the backend by calling the update user route. WDYT?

Reply:
Yes I think it's a good idea because it would allow us to be less dependent on airstack's rate limits.

Reply:
I will keep this for now. Once the backend is ready I will update the front and remove these routes